src/main/java/com/documents/controllers/StudentController.java

```diff
@@ -29,4 +40,65 @@
             return new ResponseEntity<Student>(HttpStatus.NOT_FOUND);
         }
     }
+
+    /**
+     * Return all the students from database
+     *
+     * @return
+     */
+    @RequestMapping(value = "/student/all", method = RequestMethod.GET)
```

Comment:
We could add the @RequestMapping(value = "/student") before the beggining of the class in order to avoid writing this path all over again.